lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -51,6 +51,7 @@ def _transverse_mercator_cube(self, ellipsoid=None):
         cube.add_dim_coord(coord, 1)
         return cube
 
+
```

Comment:
Whoops! I think this line is tripping up the code compatibility check. Remove it and all the tests should pass :smile: